website/docs/concepts/html/events.mdx

```diff
@@ -611,7 +611,7 @@ For more information on `Closures`, see
 
 ### Using `gloo` (concise)
 
-The easier way is with `gloo`, more specifically [`gloo_events`](https://docs.rs/gloo-events/0.1.1/gloo_events/index.html)
+The easier way is with `gloo`, more specifically [`gloo::events`](https://docs.rs/gloo-events/0.1.1/gloo_events/index.html)
```

Comment:
I dont think you intended to change this as the link still points to the individual package
```suggestion
The easier way is with `gloo`, more specifically [`gloo_events`](https://docs.rs/gloo-events/0.1.1/gloo_events/index.html)
```